js/okex.js

```diff
@@ -3176,6 +3177,35 @@ module.exports = class okex extends Exchange {
         return this.parseFundingRate (entry, market);
     }
 
+    async fetchFundingHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'type': 8,
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privateGetAccountBills (this.extend (request, params));
+        const data = this.safeValue (response, 'data');
+        const result = [];
+        for (let i = 0; i < data.length; i++) {
+            const entry = data[i];
+            const timestamp = this.safeTimestamp (entry, 'ts');
+            const instId = this.safeString (entry, 'instId');
+            const market = this.safeMarket (instId);
+            result.push ({
+                'info': entry,
+                'symbol': market['symbol'],
+                'code': market['inverse'] ? market['base'] : market['quote'],
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'id': this.safeNumber (entry, 'billId'),
+                'amount': this.safeNumber (entry, 'sz'),
+            });
+        }
+        return result;
```

Comment:
After parsing these results we must also add the filtering by symbol, if it was defined.
@samgermain if possible, please add that, as shown, for example, here:

- https://github.com/ccxt/ccxt/blob/master/js/okex3.js#L2864

Reply:
Also, filter them by `since` + `limit`.